setup.py

```diff
@@ -46,6 +46,7 @@ def get_dist(pkgname):
     'six',
     'jsonpatch',
     'websocket-client',
+    'networkx'
```

Comment:
well, since you add networkx into the requirements list you don't need to do
```python
        try:
            import networkx as nx
        except:
            raise RuntimeError(
                "networkx must be installed to plot Graph figures")
```
instead, import networkx in the file header.

But again, someone will fix this in the upcoming pull requests on this matter.

If I were a maintainer, I'd already start splitting the requirements into hard requirements and requirements-extra.